flipperdevices/flipperzero-firmware
# [FL-1247] Fix ibutton read cli command

# What's new

Fix invalid CRC print

# Verification 

- Build and upload to f5 target
- Verify correct output of tm read CLI command for keys with valid and invalid CRC

# Checklist (do not modify)
